### PR TITLE
chore(package.xml): bump version to 0.39.0

### PR DESCRIPTION
**Pull request by youtalk**
## Description

This PR bumps `humble` version to `0.39.0`
https://github.com/youtalk/autoware_launch/compare/release-0.39.0...tier4:autoware_launch:v0.39.0

See the `0.38.0` PR https://github.com/autowarefoundation/autoware_launch/pull/1226.

Ref https://github.com/autowarefoundation/autoware.universe/pull/9435

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,49 @@
 Changelog for package autoware_launch
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.39.0 (2024-11-25)
+-------------------
+* youtalk username
+* Merge commit '9d0e7055a' into release-0.39.0
+* feat: change the old diagnostic_convertor to scenario_simulator_v2_adapter (`#1227 <https://github.com/youtalk/autoware_launch/issues/1227>`_)
```
Comment (mitsudome-r) [inline autoware_launch/CHANGELOG.rst:9]: Links in CHANGELOG must be pointing to autowarefoundation

```suggestion
* feat: change the old diagnostic_convertor to scenario_simulator_v2_adapter (`#1227 <https://github.com/autowarefoundation/autoware_launch/issues/1227>`_)
```

**Review by mitsudome-r — COMMENTED** [review]
I have confirmed that this PR:
- syncs humble branch with main branch up to 9d0e7055ac59088197d1c30488ea270fe865df55
- modifies package.xml to update the version
- update CHANGELOG accordingly

once the links in CHANGELOG files are fixed, I can approve the PR.

File: autoware_launch/CHANGELOG.rst (hunk 1 above)
Comment (youtalk) [reply autoware_launch/CHANGELOG.rst]: Oh my gosh. Thank you.

Comment (mitsudome-r) [reply autoware_launch/CHANGELOG.rst]: @youtalk 

Sorry, I should've mentioned in my previous comment.
You should fix all the other links in your PR as well. Many of them in the auto-generated changelog seems to be pointing to your fork.

Comment (youtalk) [reply autoware_launch/CHANGELOG.rst]: I'm sorry but fixed. https://github.com/autowarefoundation/autoware_launch/pull/1248/commits/b292c2cff391dc4bfdacefe4bc8e774e21aa2e8e

**Review by mitsudome-r — APPROVED** [review]

**Pull request merged**